onnx/backend/test/case/node/lstm.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class LSTM(Base):
+
+    @staticmethod
+    def export_defaults():
+        input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+        input_size = 2
+        hidden_size = 4
+
+        node = onnx.helper.make_node(
+            'LSTM',
+            inputs=['X', 'W', 'R'],
+            outputs=['Y'],
+            hidden_size=hidden_size
+        )
+
+        W = np.ones((1, 4 * hidden_size, input_size)).astype(np.float32)
+        R = np.ones((1, 4 * hidden_size, hidden_size)).astype(np.float32)
+
+        output = np.array([[0.70377535, 0.70377535, 0.70377535, 0.70377535],
+                           [0.96009213, 0.96009213, 0.96009213, 0.96009213],
+                           [0.99451119, 0.99451119, 0.99451119, 0.99451119]]).astype(np.float32)
+
+        expect(node, inputs=[input, W, R], outputs=[output], name='test_lstm_defaults')
+
+    @staticmethod
+    def export_initial_bias():
+        input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+        input_size = 2
+        hidden_size = 4
+        custom_bias = 0.1
+
+        node = onnx.helper.make_node(
+            'LSTM',
+            inputs=['X', 'W', 'R', 'B'],
+            outputs=['Y'],
+            hidden_size=hidden_size
+        )
+
+        W = np.ones((1, 4 * hidden_size, input_size)).astype(np.float32)
+        R = np.ones((1, 4 * hidden_size, hidden_size)).astype(np.float32)
+
+        # Adding custom bias
+        W_B = custom_bias * np.ones((1, 4 * hidden_size)).astype(np.float32)
+        R_B = np.zeros((1, 4 * hidden_size)).astype(np.float32)
+        B = np.concatenate((W_B, R_B), 1)
+
+        output = np.array([[0.70919698, 0.70919698, 0.70919698, 0.70919698],
+                           [0.96049958, 0.96049958, 0.96049958, 0.96049958],
+                           [0.99456745, 0.99456745, 0.99456745, 0.99456745]]).astype(np.float32)
+
+        expect(node, inputs=[input, W, R, B], outputs=[output], name='test_lstm_with_initial_bias')
+
+    @staticmethod
+    def export_peepholes():
+            input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+            input_size = 2
```

Comment:
A (few) different combination of input_size and hidden_size would be good to have.

Reply:
Sure, will add.